apps/server/src/controllers/pingController.ts

```diff
@@ -0,0 +1,7 @@
+import { Request, Response } from 'express';
+
+import pkg from '../../package.json' assert { type: 'json' };
```

Comment:
this is the second import of the manifest, we could abstract it into a reusable manifest.js file that does this import and exports a simplified js object with only the variables we may want to destructure elsewhere